shell/probe/install.ps1

```diff
@@ -30,7 +32,7 @@ function LogMessage {
 
 function RegisterEndpoint {
     LogMessage "Provisioning Detect Endpoint Token..."
-    $data = @{"id"=$endpointId;"tag"="windows"} | ConvertTo-Json
+    $data = @{"id"=$endpointId;"tags"=$endpointTags + ,"windows"} | ConvertTo-Json
```

Comment:
`,` on the wrong side of the double quote

Reply:
it's actually on the correct side! it's dumb but when $endpointTags is empty, the comma there will make sure that powershell interprets "windows" as an array not a string. and when $endpointTags is not empty, then it's just adding to that list

Reply:
🤯